docs/conf.py

```diff
@@ -253,23 +253,27 @@
 # (source start file, target name, title, author,
 #  dir menu entry, description, category)
 texinfo_documents = [
-  ('index', 'zopecontainer', 'zope.container Documentation',
-   'Zope Foundation and Contributors', 'zopecontainer',
-   'Interfaces for object container components, and implementations such as a BTreeContainer and OrderedContainer',
-   'Miscellaneous'),
+    ('index',
+     'zopecontainer',
+     'zope.container Documentation',
+     'Zope Foundation and Contributors',
+     'zopecontainer',
+     'Interfaces for object container components, and implementations '
+     'such as a BTreeContainer and OrderedContainer',
```

Comment:
```suggestion
     'Interfaces for object container components, and implementations'
     ' such as a BTreeContainer and OrderedContainer',
```
Improving readability: make it clear that the second line continues the first one without havong to look at the line end.